apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -18,8 +18,8 @@ const RecentCourses = React.createClass({
 
   render() {
     const { courses, showAllCoursesLink, isTeacher, heading, isRtl } = this.props;
-    const topFourCourses = courses.length > 4 ? courses.slice(0,4) : courses;
-    const moreCourses = courses.length > 4 ? courses.slice(4) : [];
+    const topFourCourses = courses.length >= 4 ? courses.slice(0,4) : courses;
+    const moreCourses = courses.length > 4 ? courses.slice(4, -1) : [];
```

Comment:
What does the -1 do here?  I think it chooses elements through to the second-last item in the array.